orderbook/src/api/filter.rs

```diff
@@ -80,6 +88,41 @@ pub mod test_util {
         assert_eq!(response_orders, orderbook_orders);
     }
 
+    #[tokio::test]
+    async fn get_order_by_uid_() {
+        let orderbook = Arc::new(OrderBook::default());
+        let filter = get_order_by_uid(orderbook.clone());
+        let mut order_creation = OrderCreation::default();
+        order_creation.valid_to = u32::MAX;
+        order_creation.sign_self();
+        let order = orderbook.order_creation_to_order(order_creation).unwrap();
+        orderbook.add_order(order_creation).await.unwrap();
```

Comment:
This function already returns the uid so there is no need to manually to `order_creation_to_order` here and that function can be made private again (this PR makes it `pub` so that you can use it here).